src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -681,72 +672,11 @@ private function addServices()
             } else {
                 $privateServices .= $this->addService($id, $definition);
             }
-
-            $synchronizers .= $this->addServiceSynchronizer($id, $definition);
```

Comment:
The `$synchronizers` variable can also be removed as its not really used anymore now.